docs/EN_US/ContainerizedHPCC/ContainerizedMods/RemoteAccess.xml

```diff
@@ -0,0 +1,347 @@
+        </listitem>
+
+        <listitem>
+          <para>Add the jetstack repo and install the jetstack certificate
+          manager.</para>
+
+          <para><programlisting>helm repo add jetstack https://charts.jetstack.io
+helm repo update
+helm install cert-manager jetstack/cert-manager --version v1.7.1 --set installCRDs=true</programlisting></para>
+        </listitem>
+
+        <listitem>
+          <para>Set <emphasis role="bold">certificates.enabled</emphasis> to
+          <emphasis>true</emphasis> in a custom values yaml file.</para>
+
+          <para><programlisting>certificates:
+  <emphasis role="red">enabled: true</emphasis></programlisting></para>
+
+          <variablelist>
+            <varlistentry>
+              <term>Note:</term>
+
+              <listitem>
+                <para>This enables the use of certificates for
+                <emphasis>all</emphasis> esp services and enables mutual TLS
+                (mTLS) between components, too. If you don’t want that (for
+                example, for testing reasons), and only want certificates
+                enabled for the ESP DFS service, then set enabled: false for
+                all other certificates.issuers (such as, "local", "public",
+                "signing", etc.)</para>
+              </listitem>
+            </varlistentry>
+          </variablelist>
+        </listitem>
+
+        <listitem>
+          <para>Set <emphasis
+          role="bold">certificates.issuers.remote.enabled</emphasis> to
+          <emphasis>true</emphasis> in the custom values yaml file.</para>
+
+          <para><programlisting>remote:
+    name: hpcc-remote-issuer
+    <emphasis role="red">enabled: true</emphasis>
+    kind: Issuer
+    spec:
+      ca:
+        secretName: hpcc-remote-issuer-key-pair</programlisting></para>
+        </listitem>
+
+        <listitem>
+          <para>In the custom values yaml file, add a <emphasis
+          role="bold">remoteClients:</emphasis> section to the Esp <emphasis
+          role="bold">dfs</emphasis> service for each client environment that
+          will connect.</para>
+
+          <para><programlisting>- name: dfs
+  application: dfs
+  auth: none
+  replicas: 1
+  <emphasis role="red">remoteClients:
+  - name: env2
+    organization: mycompany</emphasis>
+  service:
+    visibility: local
+    servicePort: 8520
+  resources:
+    cpu: "250m"
+    memory: "1G"</programlisting></para>
+        </listitem>
+
+        <listitem>
+          <para>Create a root certificate and private key for the <emphasis
+          role="bold">remote</emphasis> certificates issuer authority</para>
+
+          <para><variablelist>
+              <varlistentry>
+                <term>Note:</term>
+
+                <listitem>
+                  <para>The following example command should be entered on one
+                  line without the "\" connectors. The connectors are only
+                  included here so the code will fit within page
```

Comment:
That's not true in Linux shells.. the backslashes are line continuations, you can copy and paste this multiline command inc. backslashes and it will work. So I'd either remove this text or adjust if you know it doesn't work in some scenarios.

Or perhaps, to avoid confusion, remove but change the command to a single line, so works in all scenarios.

Reply:
In Docbook, using our XSLT, the words run off the page in the PDF if they are on one line. In my Linux shell (Ubuntu 20.04 in Windows terminal), pasting with the slashes doesn't work.